ai2thor/tests/test_unity.py

```diff
@@ -64,7 +61,8 @@ def test_small_aspect():
     event = controller.step(dict(action='Initialize', gridSize=0.25))
     assert event.frame.shape == (64, 128, 3)
     
-def test_lookdown():
+@pytest.mark.parametrize("controller", [wsgi_controller, fifo_controller])
```

Comment:
nice

Reply:
Yeah, I just discovered that parametrize for this.  Its pretty nice.